frontend/public/components/utils/telemetry.ts

```diff
@@ -1,5 +1,5 @@
 import { Location } from 'history';
-import { getBrandingDetails } from '../masthead';
+import { getBrandingDetails } from '../masthead-branding';
```

Comment:
Even more reason to move this file out of the components folder and rename to just branding.ts. It's used outside of the masthead component.